src/quo2/components/community/discover_card.cljs

```diff
@@ -0,0 +1,94 @@
+                :color               (colors/theme-colors
+                                      colors/neutral-50
+                                      colors/neutral-40)
+                :weight               :regular
+                :size                 :paragraph-2}
+     description]]])
+
+(defn placeholder-list-images [{:keys [images width height border-radius]}]
+  [react/view
+   [react/view {:justify-content    :center}
+    (for [{:keys [id]} images]
+      ^{:key id}
+      [react/view {:border-radius    border-radius
+                   :margin-top       4
+                   :margin-right     4
+                   :width            width
+                   :height           height
+                   :background-color colors/neutral-10}])]])
+
+(defn placeholder-row-images [{:keys [first-image last-image images width height
+                                      border-radius]}]
+  [react/view
+   (when first-image
+     [react/view {:border-bottom-right-radius 6
+                  :border-bottom-left-radius  6
+                  :margin-right               4
+                  :width                      width
+                  :height                     height
+                  :background-color           colors/neutral-10}])
+   (when images
+     [placeholder-list-images {:images        images
+                               :width         32
+                               :height        32
+                               :border-radius 6}])
+   (when last-image
+     [react/view {:border-top-left-radius    border-radius
+                  :border-top-right-radius   6
+                  :margin-top                4
+                  :width                     width
+                  :height                    height
+                  :background-color          colors/neutral-10}])])
+
+(defn discover-card [{:keys [title description on-press]}]
+  (let [on-joined-images     (get images   :images)]
+    [react/touchable-without-feedback
+     {:on-press on-press}
+     [react/view (merge (styles/community-card 16)
+                        {:background-color  (colors/theme-colors
+                                             colors/white
+                                             colors/neutral-90)}
+                        {:flex-direction  :row
+                         :margin-horizontal 20
+                         :height          56
+                         :padding-right   12})
+      [card-title-and-description title description]
+      (for [{:keys [id column-images]} on-joined-images]
+        ^{:key id}
+        [placeholder-row-images {:images        (when (= id 1)
+                                                  column-images)
+                                 :width         32
+                                 :height        (if (= id 1) 8 26)
+                                 :border-radius 6
+                                 :first-image   "" ;; TODO replace with real data
```

Comment:
[Code style] Quick one, try to use a single semicolon for end of line comments and reserve double semicolons for line comments only.